packages/shared/src/components/squads/SharePostBar.tsx

```diff
@@ -17,13 +22,15 @@ export type NewSquadPostProps = Pick<
 interface SharePostBarProps {
   className?: string;
   onNewSquadPost?: (props?: NewSquadPostProps) => void;
+  squad: Squad;
```

Comment:
Since this is like an input component for us as a whole, we can probably just accept a `disabled` boolean property than a whole object. Then we can proceed with checking whether to return the placeholder or not. WDYT?

Reply:
Great idea ✅